lib/go/test/flow_idtable_staking_test.go

```diff
@@ -1838,7 +1919,7 @@ func TestIDTable(t *testing.T) {
 
 		tx := flow.NewTransaction().
 			SetScript(templates.GenerateEndStakingScript(env)).
-			SetGasLimit(100).
+			SetGasLimit(10000).
```

Comment:
why increasing it?

How do I know what's the proper gas limit to use for a transaction?

Reply:
I increased the gas limit because my tests had made it so 100 limit wasn't enough. I don't know what the actual amount it uses is, though. I just increased it to an arbitrary number